data_steward/curation_logging/curation_gae_handler.py

```diff
@@ -137,10 +134,7 @@ def get_highest_severity_level_from_lines(lines):
     """
     if lines:
 
-        s = sorted(
-            [line['severity'] for line in lines],
-            key=lambda severity: -severity
-        )
+        s = sorted([line['severity'] for line in lines], key=lambda severity: -severity)
```

Comment:
Again, google style is more aggressive with filling up the line length.  pep-8 style transforms lines 140-143 to:
```
s = sorted([line['severity'] for line in lines],
                   key=lambda severity: -severity)
```
I feel like the pep-8 style is more readable.  each argument is on its own line.

Reply:
This is also a by product of extending the line length.